src/plexil-adapter/OwInterface.h

```diff
@@ -111,15 +134,16 @@ class OwInterface
 
   template <typename T>
   bool checkFaultMessages(std::string fault_component,
-                                        T msg_val,
-                                        std::string key,
-                                        T value,
-                                        bool b );
-
-//////////////////// FAULTS FOR SYSTEM LEVEL ////////////////////////
-// structure of all maps for faults is the following:
-// key = (string) fault name
-// value = (pair) ( (int) numberical fault value, booleon of if fault exists)
+                          T msg_val,
+                          std::string key,
+                          T value,
+                          bool b );
```

Comment:
Maybe it makes sense to pass string values as const ref since you are not modifying them inside the method

Reply:
Lannsie's addition, but I didn't catch that.  Changed.  However this function is deleted in my other branch!